# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr-text2skin/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- text2skin/cache.h.orig	2005-01-31 15:40:29.000000000 +0100
+++ text2skin/cache.h	2008-11-02 21:50:59.000000000 +0100
@@ -19,21 +19,21 @@
 
 	item_map   mItems;
 	usage_list mUsage;
-	uint       mMaxItems;
+	unsigned int       mMaxItems;
 
 protected:
 	virtual void DeleteObject(const key_type &Key, data_type &Data) = 0;
 	virtual void ResetObject(data_type &Data) = 0;
 
 public:
-	cxCache(uint MaxItems);
+	cxCache(unsigned int MaxItems);
 	virtual ~cxCache();
 
 	void Reset(void);
 	void Flush(void);
 	bool Contains(const key_type &Key);
 	data_type &operator[](const key_type &Key);
-	uint Count(void) { return mUsage.size(); }
+	unsigned int Count(void) { return mUsage.size(); }
 };
 
 template<class key_type, class data_type>
@@ -43,7 +43,7 @@
 }
 
 template<class key_type, class data_type>
-cxCache<key_type, data_type>::cxCache(uint MaxItems) 
+cxCache<key_type, data_type>::cxCache(unsigned int MaxItems) 
 {
 	mMaxItems = MaxItems;
 }
